packages/common/src/resourceIdToHex.ts

```diff
@@ -0,0 +1,23 @@
+import { Hex, stringToHex, concatHex } from "viem";
+import { ResourceId } from "./common";
+import { ResourceType } from "./resourceTypes";
+
+/** @internal */
```

Comment:
ooc what does `@internal` do?

Reply:
ideally keeps it from being exported outside of the package: https://typedoc.org/tags/internal/